app/models/manageiq/providers/openstack/inventory/collector/target_collection.rb

```diff
@@ -191,6 +191,14 @@ def vms_by_id
     @vms_by_id ||= Hash[vms.collect { |s| [s.id, s] }]
   end
 
+  def cloud_volumes
```

Comment:
we need the same for cloud_volume_backups and cloud_volume_snapshots, otherwise the targeted refresh would be deleting those (because they would not be found in in the API)